democracylab_environment_variables.sh

```diff
@@ -37,7 +37,7 @@ export POSITION_DESCRIPTION_EXAMPLE_URL='https://docs.google.com/document/d/142N
 export VOLUNTEER_CONCLUDE_SURVEY_URL='https://docs.google.com/forms/d/e/1FAIpQLSd4e9FQYX3ARPg7qz1ct5qM_bQW_kkEvQaMuM5LV9Ma1NDgbA/viewform'
 
 # Configure number of projects to display per server request
-export PROJECTS_PER_PAGE=20
+export PROJECTS_PER_PAGE=2
```

Comment:
This isn't a good default unless you're specifically testing pagination, so best to keep it at 20 I think.

Reply:
Oops